README.md

```diff
@@ -17,10 +17,7 @@
 | [cc-dynamic](./images/cc-dynamic) | `cgr.dev/chainguard/cc-dynamic` | deprecated | [![](https://storage.googleapis.com/chainguard-images-build-outputs/badges/cc-dynamic.build.status.latest.svg)](https://registry-ui.chainguard.app/?image=cgr.dev/chainguard/cc-dynamic:latest) |
-| [cert-manager-acmesolver](./images/cert-manager-acmesolver) | `cgr.dev/chainguard/cert-manager-acmesolver` | experimental | [![](https://storage.googleapis.com/chainguard-images-build-outputs/badges/cert-manager-acmesolver.build.status.latest-dev.svg)](https://registry-ui.chainguard.app/?image=cgr.dev/chainguard/cert-manager-acmesolver:latest-dev)<br/>[![](https://storage.googleapis.com/chainguard-images-build-outputs/badges/cert-manager-acmesolver.build.status.latest.svg)](https://registry-ui.chainguard.app/?image=cgr.dev/chainguard/cert-manager-acmesolver:latest) |
-| [cert-manager-cainjector](./images/cert-manager-cainjector) | `cgr.dev/chainguard/cert-manager-cainjector` | experimental | [![](https://storage.googleapis.com/chainguard-images-build-outputs/badges/cert-manager-cainjector.build.status.latest-dev.svg)](https://registry-ui.chainguard.app/?image=cgr.dev/chainguard/cert-manager-cainjector:latest-dev)<br/>[![](https://storage.googleapis.com/chainguard-images-build-outputs/badges/cert-manager-cainjector.build.status.latest.svg)](https://registry-ui.chainguard.app/?image=cgr.dev/chainguard/cert-manager-cainjector:latest) |
-| [cert-manager-controller](./images/cert-manager-controller) | `cgr.dev/chainguard/cert-manager-controller` | experimental | [![](https://storage.googleapis.com/chainguard-images-build-outputs/badges/cert-manager-controller.build.status.latest-dev.svg)](https://registry-ui.chainguard.app/?image=cgr.dev/chainguard/cert-manager-controller:latest-dev)<br/>[![](https://storage.googleapis.com/chainguard-images-build-outputs/badges/cert-manager-controller.build.status.latest.svg)](https://registry-ui.chainguard.app/?image=cgr.dev/chainguard/cert-manager-controller:latest) |
-| [cert-manager-webhook](./images/cert-manager-webhook) | `cgr.dev/chainguard/cert-manager-webhook` | stable | [![](https://storage.googleapis.com/chainguard-images-build-outputs/badges/cert-manager-webhook.build.status.latest-dev.svg)](https://registry-ui.chainguard.app/?image=cgr.dev/chainguard/cert-manager-webhook:latest-dev)<br/>[![](https://storage.googleapis.com/chainguard-images-build-outputs/badges/cert-manager-webhook.build.status.latest.svg)](https://registry-ui.chainguard.app/?image=cgr.dev/chainguard/cert-manager-webhook:latest) |
+| [cert-manager](./images/cert-manager) | `cgr.dev/chainguard/cert-manager` | stable | [![](https://storage.googleapis.com/chainguard-images-build-outputs/badges/cert-manager.build.status.latest.svg)](https://registry-ui.chainguard.app/?image=cgr.dev/chainguard/cert-manager:latest) |
```

Comment:
this is ~wrong, but I can update `monopod` to do this properly if we're ok with this approach